release-notes/Cloud/DCP_change_log.md

```diff
@@ -123,13 +127,53 @@ When you create a DaaS system, your dataminer.services account will automaticall
 
 You can manually delete a DaaS system from dataminer.services, just like any other DMS; however, note that this is irreversible, and all data of the system will be lost.
 
+#### 28 November 2023 - Enhancement - Deploying multiple DaaS Express systems simultaniously
+
+From now on it is possible to deploy more than one DaaS Express system simultaniously on [dataminer.services](https://dataminer.services).
+
+#### 24 November 2023 - Fix - Improved username validation when deploying a DaaS Express system
+
+The username validation has been improved to prevent the deployment from failing when the given username was not a valid. This issue was introduced on 10 November 2023.
+
+#### 24 November 2023 - Enhancement - Admin app improved login system
+
+From now on the [Admin app](https://admin.dataminer.services) has an improved login system that should prevent login redirect loops that could be experienced before.
+
+#### 24 November 2023 - New feature - Introducing the new Catalog user interface
+
+From now on [catalog.dataminer.services](https://catalog.dataminer.services) will have a completely renewed user interface.
+
+#### 22 November 2023 - Enhancement - Warning to link account before doing a DxM or Catalog deployment
+
+From now on the [Admin app](https://admin.dataminer.services) and [Catalog](https://catalog.dataminer.services) will prevent users from doing a deployment if they don't have a linked DataMiner account for the particular DataMiner System because the deployment would fail anyhow. A warning to link your accounts will be displayed instead.
+
+#### 22 November 2023 - Enhancement - dataminer.services homepage improved login system
+
+From now on [dataminer.services](https://dataminer.services) has an improved login system that should prevent login redirect loops that could be experienced before.
+
 #### 21 November 2023 - New feature - Remote access using DataMiner Cube [ID_37841]
 
 It is now possible to connect to a DataMiner System via remote access using DataMiner Cube. If remote access is enabled for a DMS and you have been granted access to dataminer.services features, you can access the DMS remotely via Cube using the same URL as for remote access to the web pages, but without the protocol prefix `https://`. A button is also available on dataminer.services and in the Admin app that can be used to open Cube with the correct remote access filled in as the host.
 
 > [!NOTE]
 > At present, there is still a limitation to this feature: if the DMS has SAML authentication configured, users will not be able to access the DMS remotely with Cube.
 
+#### 10 November 2023 - Fix - Grant admin consent linking button unresponsive
+
+The grant admin consent linking button wouldn't work anymore. This issue has now been resolved.
```

Comment:
```suggestion
Users were no longer able to grant admin consent for the Teams Chat Integration because the button was no longer functional. This issue has now been resolved.
```